### ISSUE TITLE
"appearance" doesn't save

### ISSUE THREAD

**Issue by eksynn** [XeTrinityz/ThatSkyApp#3]
appearance tab doesn't get saved when saving settings.
that includes: avatar height, avatar scale (not sure what the difference is), fake cape level
(i didn't test the shout editor enough so i can't tell if it gets saved enough)

**Comment by XeTrinityz** [comment]
It wouldn't make sense to have these options save.

**Comment by eksynn** [comment]
so i don't have to set it up again when i log in every time xD .. at least it made sense to me ( ╥ ᴗ╥)

**Comment by XeTrinityz** [comment]
Other players can not see these changes

**Comment by eksynn** [comment]
oh.. i didn't know xD yea in that case it wouldn't really make sense for it to save. aw man

**Comment by XeTrinityz** [comment]
No, unfortunately not as I believe it would actually be a hindrance as this would require the user to manually remove the saved option from their configuration file in order to reset the values to their original states.

**Comment by eksynn** [comment]
that is too bad.

... what's the name of the permanent height change spell? the one that randomises whether you go higher or lower - if i can maybe use that, then i can just be permanently tiny xD
